src/bindings/python/src/pyopenvino/utils/utils.cpp

```diff
@@ -47,6 +48,8 @@ PY_TYPE check_list_element_type(const py::list& list) {
             check_type(PY_TYPE::BOOL);
         } else if (py::isinstance<ov::PartialShape>(it)) {
             check_type(PY_TYPE::PARTIAL_SHAPE);
+        } else if (py::isinstance<ov::hint::ModelDistributionPolicy>(it)) {
+            check_type(PY_TYPE::ModelDistributionPolicy);
```

Comment:
What about other `ov::hint` types if in the container?

Reply:
Now there is no other `ov::hint` type in this container.